test/unit/test_helpers_tests.rb

```diff
@@ -50,6 +50,24 @@ class UnitTests < Assert::Context
       assert_equal exp, subject.params[Dk::ABDeploy::RELEASE_B_DIR_PARAM_NAME]
     end
 
+    should "setup the params the update task does" do
+      subject.set_dk_abdeploy_validate_params
```

Comment:
@kellyredding - Since `Update` runs `Validate` as a before callback you might make the `set_dk_abdeploy_update_params` automatically call the validate params helper so you don't have to do both. Having to call the validate params helper feels like having to know internals that I shouldn't be worried about.